stubdefaulter.py

```diff
@@ -305,6 +305,12 @@ def main() -> None:
         ),
         default=(),
     )
+    parser.add_argument(
+        "-z",
+        "--exit-zero",
+        action="store_true",
+        help="Always exit with code 0, even if there were errors.",
```

Comment:
```suggestion
        help="Exit with code 0 even if there were errors.",
```
It's not "always" because we might crash.